modules/ROOT/pages/create-custom-namespace.adoc

```diff
@@ -12,13 +12,14 @@ When you deploy an application into an Anypoint Platform environment, Runtime Fa
 [source,copy]
 ----
 kubectl create ns <YOUR_ENVIRONMENT_NAME>
-kubectl label ns <YOUR_ENVIRONMENT_NAME> rtf.mulesoft.com/role=workers rtf.mulesoft.com/envId=<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID>
+kubectl label ns <YOUR_ENVIRONMENT_NAME> rtf.mulesoft.com/agentNamespace=rtf rtf.mulesoft.com/role=workers rtf.mulesoft.com/envId=<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID> rtf.mulesoft.com/org=<YOUR_ANYPOINT_PLATFORM_ORGANIZATION_ID>
```

Comment:
Instead of `rtf` we should probably be adding a placeholder for the namespace `<rtf_namespace>`